src/index.html

```diff
@@ -6,9 +6,49 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html" class="header__logo_link">
+        <img src="./images/logo.png"
+            alt="moyoLogo"
+            width="40px"
+            height="40px">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav_items">
+          <li class="header__nav_item">
```

Comment:
Fix classes everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
```